generators/app/index.js

```diff
@@ -581,6 +581,16 @@ module.exports = yeoman.Base.extend({
         }
       }
     },
+    setLog4jdevProperties: function () {
+      if (this.bail) return;
+
+      // set the dev-log4j.properties file for the project package
+      var devLog4jPropertiesPath = this.destinationPath('repo/src/main/resources/alfresco/extension/dev-log4j.properties');
```

Comment:
Sorry for the swicheroo, maybe just use a template with context for this rather than having it all inline?